### PR TITLE
Update to the new Firebase Crashlytics

### PR DESCRIPTION
**Pull request by Thomas-Vos**
This PR updates Crashlytics to the new Firebase Crashlytics. The old Fabric Crashlytics is deprecated and will stop working soon. I updated the README and the sample app, let me know if I missed something.

Closes https://github.com/touchlab/CrashKiOS/issues/3

https://firebase.google.com/docs/crashlytics/get-started?platform=ios
> Note: The Fabric SDK is now deprecated and will continue reporting your app's crashes until November 15, 2020. On this date, the Fabric SDK and old versions of the Firebase Crashlytics SDK will stop sending crashes for your app. To continue getting crash reports in the Firebase console, make sure you upgrade to the Firebase Crashlytics SDK versions 17.0.0+ for Android, 4.0.0+ for iOS, and 6.15.0+ for Unity.

### CODE REVIEW

File: sample/iosAppCrashlytics/Podfile.lock [hunk 1]
```diff
@@ -1,115 +1,115 @@
...
-  - Fabric
   - Firebase/Analytics
+  - Firebase/Crashlytics
   - sample (from `..`)
 
 SPEC REPOS:
-  https://github.com/CocoaPods/Specs.git:
-    - Crashlytics
-    - Fabric
+  trunk:
```
Comment (Thomas-Vos) [inline sample/iosAppCrashlytics/Podfile.lock:82]: I'm not really sure why this "spec repo" changed so if I have something wrong in my cocoapods configuration please let me know.

**Comment by vaudevillen** [comment]
does this also need an update to the build phase run script?
Something like this, that's in the current directions? 
```
{PATH_TO_PODS}/FirebaseCrashlytics/upload-symbols -gsp {PATH_TO_GOOGLESERVICE_PLIST} -p ios {PATH_TO_SHARED_MODULE_DEBUG_FRAMEWORK_DSYM}
{PATH_TO_PODS}/FirebaseCrashlytics/upload-symbols -gsp {PATH_TO_GOOGLESERVICE_PLIST} -p ios {PATH_TO_SHARED_MODULE_RELEASE_FRAMEWORK_DSYM}
```

**Comment by vaudevillen** [comment]
oh whoops, missed that part. thanks!

File: sample/iosAppCrashlytics/Podfile.lock (hunk 1 above)
Comment (nikolaykasyanov) [reply sample/iosAppCrashlytics/Podfile.lock:82]: That's totally fine, at some point CocoaPods switched to the new "trunk" repo: https://blog.cocoapods.org/CocoaPods-1.8.0-beta/.

**Comment by nikolaykasyanov** [comment]
Thanks for this PR! Would be nice to get it merged upstream 🙏

**Comment by kpgalligan** [comment]
Sorry for delay. We moved most of the "public" support in the library to Kermit, although we may revisit making CrashKiOS somewhat more supported for people who don't want the rest of Kermit's logging.

**Pull request merged**